File: lib/starter3.py
class Team:
    def __init__(self):
        self.name = "none"
        self.wins = 0
        self.losses = 0

    # Define get_win_percentage()
    def get_win_percentage(self):
        wins = float(self.wins)
        return wins / (wins + float(self.losses))

    # Define print_standing()
    def print_standing(self):
        win_percentage = self.get_win_percentage()
        print(f"Win percentage: {win_percentage:.2f}")
        if win_percentage >= 0.5:
            print(f"Congratulations, Team { self.name } has a winning average!")
        else:
            print(f"Team { self.name } has a losing average.")


if __name__ == "__main__":
    team = Team()

    user_name = input()
    user_wins = int(input())
    user_losses = int(input())

    team.name = user_name
    team.wins = user_wins
    team.losses = user_losses

    team.print_standing()
  

File: lib/starter3.rb
class Team
  attr_accessor :name, :wins, :losses
  def initialize(name = "none", wins = 0, losses = 0)
    @name = name
    @wins = wins
    @losses = losses
  end

  def get_win_percentage
    wins = @wins.to_f
    wins/ (wins + @losses.to_f)
  end

  def print_standing
    win_percentage = get_win_percentage
    puts("Win percentage: %.2f" % win_percentage)
    if win_percentage >= 0.5
      puts("Congratulations, Team #{@name} has a winning average!")
    else
      puts("Team #{@name} has a losing average.")
    end
  end
end
